Add doc dir for Keptn 0.16 (#1228)
keptn/keptn.github.io: content/docs/0.16.x/operate/api_token/index.md

    <p>

    * To generate an API token, use the following command and store it into the environment variable `KEPTN_API_TOKEN`:

        ```console
        KEPTN_API_TOKEN=$(LC_CTYPE=C tr -dc "a-zA-Z0-9" < /dev/urandom | head -c 45)
        ```

    * To create an API token, execute:

        ```console
        kubectl create secret generic -n keptn keptn-api-token --from-literal=keptn-api-token="$KEPTN_API_TOKEN"
        ```

    </p>
    </details>

    <details><summary>on Windows PowerShell</summary>
    <p>

    * To generate an API token, use the following command and store it into the environment variable `$Env:KEPTN_API_TOKEN`:

        ```console
        $Env:KEPTN_API_TOKEN =  Write-Output ( -join ((0x30..0x39) + ( 0x41..0x5A) + ( 0x61..0x7A) | Get-Random -Count 45  | % {[char]$_}) )
        ```

    * To create an API token, execute:

        ```console
        kubectl create secret generic -n keptn keptn-api-token --from-literal=keptn-api-token="$Env:KEPTN_API_TOKEN"
        ```

    </p>
    </details>

If you want to make use of a self provided API token during the installation of Keptn, you can use the  `tokenSecretName` helm value.

> This will prevent Helm from generating a new secret on installation and instead will have helm use the secret you provided.

Reviewer: ```suggestion
This prevents Helm from generating a new secret on installation and instead has Helm use the secret you provided.
```

## Reset API Token

* To reset an API token of a Keptn installation, first delete the secret:

    ```console
    kubectl delete secret -n keptn keptn-api-token
    ```

* [Create a new API token](./#create-api-token) as explained above.

* Restart API service since it requires the new token:

    ```console
    kubectl delete pods -n keptn --selector=app.kubernetes.io/name=api-service
    ```

* Re-authenticate Keptn CLI as explained [here](../../reference/cli/commands/keptn_auth).

> * **Don't forget** to replace the API token at tools that interact with the Keptn API, e.g., *Problem Notification* setting in Dynatrace or *WebHook* in Prometheus.

Reviewer: ```suggestion
* **Don't forget** to replace the API token for tools that interact with the Keptn API, e.g., *Problem Notification* setting in Dynatrace or *WebHook* in Prometheus.
```
